promql/promqltest/test.go

```diff
@@ -978,6 +1125,16 @@ func (ev *evalCmd) checkExpectedFailure(actual error) error {
 		}
 	}
 
+	expFail, exists := ev.expectedCmds[Fail]
+	if exists && len(expFail) > 0 {
+		if len(expFail) > 1 {
+			return fmt.Errorf("expected more than one error evaluating query %q (line %d), but got only one", ev.expr, ev.line)
+		}
```

Comment:
I think we should just fail to parse any test with multiple failure expectations: I don't think that's a well-formed test case.